### ISSUE TITLE
user without admin permissions can handle plugin configurations? 

### ISSUE THREAD

**Issue by mredodos** [eternal-flame-AD/gotify-broadcast#11]
Hi im try your plugin but i dont understand if is normal an user without admin permissions can edit plugin configurations

**Comment by eternal-flame-AD** [comment]
Yes, every body get their own filters they can edit in the WebUI. :)

**Comment by mredodos** [comment]
It's probably fatigue, but I don't really understand how it works. 

then I can activate it on my admin account and send a message to that channel but how do I subscribe other non-admin users to the same channel?

**Comment by eternal-flame-AD** [comment]
You don't need to subscribe, if you do not have sender_filter than rejects this message and the receiver dont have a receiver_filter that rejects the message, the message will be delivered.

**Comment by mredodos** [comment]
Sorry but i dont understand how work... 

i need to put setting from one user and create a channel after i need to go on other user and set the same channel and other filters?

**Comment by eternal-flame-AD** [comment]
On the sending user, put the channel name in the `channels:`, configuration. Then you can push messages to that channel through the plugin web hook.

On the receiving user, the default is every broadcast will be received so you only need to enable the plugin and your broadcasts will be received. If you want to selectively receive messages you can set up the receiver filter as described.
